lib/workers/repository/process/limits.ts

```diff
@@ -7,135 +7,77 @@ import { scm } from '../../../modules/platform/scm';
 import { ExternalHostError } from '../../../types/errors/external-host-error';
 import type { BranchConfig } from '../../types';
 
-export async function getPrHourlyRemaining(
+export async function getPrHourlyCount(
   config: RenovateConfig,
 ): Promise<number> {
-  if (config.prHourlyLimit) {
-    try {
-      logger.debug('Calculating hourly PRs remaining');
-      const prList = await platform.getPrList();
-      const currentHourStart = DateTime.local().startOf('hour');
-      logger.debug(`currentHourStart=${String(currentHourStart)}`);
-      const soFarThisHour = prList.filter(
-        (pr) =>
-          pr.sourceBranch !== config.onboardingBranch &&
-          pr.sourceBranch.startsWith(config.branchPrefix!) &&
-          DateTime.fromISO(pr.createdAt!) > currentHourStart,
-      );
-      const prsRemaining = Math.max(
-        0,
-        config.prHourlyLimit - soFarThisHour.length,
-      );
-      logger.debug(`PR hourly limit remaining: ${prsRemaining}`);
-      return prsRemaining;
-    } catch (err) {
-      // istanbul ignore if
-      if (err instanceof ExternalHostError) {
-        throw err;
-      }
-      logger.error({ err }, 'Error checking PRs created per hour');
-      return config.prHourlyLimit;
+  try {
+    const prList = await platform.getPrList();
+    const currentHourStart = DateTime.local().startOf('hour');
+    logger.debug(
+      `Calculating PRs created so far in this hour currentHourStart=${String(currentHourStart)}`,
+    );
+    const soFarThisHour = prList.filter(
+      (pr) =>
+        pr.sourceBranch !== config.onboardingBranch &&
+        pr.sourceBranch.startsWith(config.branchPrefix!) &&
+        DateTime.fromISO(pr.createdAt!) > currentHourStart,
+    );
+    logger.debug(
+      `${soFarThisHour.length} PRs have been created so far in this hour.`,
+    );
+    return soFarThisHour.length;
+  } catch (err) {
+    // istanbul ignore if
+    if (err instanceof ExternalHostError) {
+      throw err;
     }
+    logger.error({ err }, 'Error checking PRs created per hour');
+    return 0;
   }
-  return Number.MAX_SAFE_INTEGER;
 }
 
-export async function getConcurrentPrsRemaining(
+export async function getConcurrentPrsCount(
   config: RenovateConfig,
   branches: BranchConfig[],
 ): Promise<number> {
-  if (config.prConcurrentLimit) {
-    logger.debug(`Calculating prConcurrentLimit (${config.prConcurrentLimit})`);
+  const openPrs: Pr[] = [];
```

Comment:
```suggestion
  const openPrCount = 0;
```

Simplify below. Same for branch count